### DISCUSSION TITLE
The command "GitHub Copilot: Enable/Disable Copilot completions" (github.copilot.toggleCopilot) is not working at all

### DISCUSSION THREAD

**Discussion by olfMombach** [community/community#112632] [Copilot Conversations]
### Select Topic Area

Bug

### Body

In a saved workspace, in all tested languages, when using a keyboard shortcut to invoke or when directly calling the command from Ctrl+Shift+P, in no way shape or form does the command "github.copilot.toggleCopilot" do anything, the extension stays enabled and giving suggestions without being able to be turned off.

Version: 1.87.1 (user setup)
Commit: 1e790d77f81672c49be070e04474901747115651
Date: 2024-03-06T00:21:36.258Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.19045Version: 1.87.1 (user setup)
Commit: 1e790d77f81672c49be070e04474901747115651
Date: 2024-03-06T00:21:36.258Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.19045

**Comment by shoddyguard** [comment]
Experiencing this too as of the latest update, I've found that disabling it for a specific language (i.e `markdown`) *does* still work, so that's at least a sort of workaround.

**Comment by Kyliathy** [comment]
I am also having this bug. It's been a long time there. 

Disabling per language is not at all convenient. It is not bindable to a hotkey. The hotkey doesn't work and completely ruins my workflow, because I get useless suggestions when I'm typing comments.

Somebody should fix this because this will also reduce costs for GitHub/Microsoft when people stop asking for completions even when they don't need them.

It's important for this to be done in a quick, easy way. Switch on/off fast at a hotkey.

**Comment by hitarth-gg** [comment]
Experiencing the same issue, Enabling/Disabling copilot globally is not working at all.

**Comment by Kyliathy** [comment]
They fixed it in latest version :). @olfMombach

**Comment by Berganzio** [comment]
Guess what. It came back. Now they removed github.copilot.toggleCopilot and there's no workaround anymore

**Reply by prism4time** [reply]
I tried pre-release version, and it fixed the problem of failing to disbale.

**Reply by SalaryTheft** [reply]
Unfortunately pre-release version didn't fixed the problem, at lease in my case.
- Version: 1.293.1471
- Build: nightly
- Editor: vscode/1.98.2

**Comment by tellscode** [comment]
Found `github.copilot.completions.toggle` in [this thread](https://github.com/microsoft/vscode-copilot-release/issues/6243), which is working for me.
